meta/documents/changelog_de.md

```diff
@@ -1,5 +1,43 @@
 # Release Notes für Ceres
 
+## v5.0.3 (2020-06-20) <a href="https://github.com/plentymarkets/plugin-ceres/compare/5.0.2...5.0.3" target="_blank" rel="noopener"><b>Übersicht aller Änderungen</b></a>
+
+### Geändert
+
+- Die Einstellung **Vorausgewählte Anrede** wurde dem Registrierungs-Widget hinzugefügt.
+- Die Performance des ShopBuilders wurde für eine größere Anzahl an Eigenschaften optimiert.
+- Auf dem umschließenden Element der Artikeleinheit auf der Artikeleinzelansicht wird nun die CSS-Klasse `.is-single-c62` verwendet, wenn es sich um die Einheit "1 Stück" handelt.
```

Comment:
Ich meine, dass es "is-single-piece" sein sollte. Kann sein, das Chris die Änderung vergessen hat ins Changelog zu packen